docs/modules/migrate/pages/dmt-connection-config.adoc

```diff
@@ -38,6 +38,33 @@ If you have multiple data structures use a new line for each map name, or use wi
 
 . Save the file
 
+NOTE: If migrating Replicated map metadata, you must add the custom JAR to your class path and restart the cluster. If this is not done, the Replicated map metadata is not migrated. For further information on adding the custom JAR to your classpath, refer to the https://docs.oracle.com/javase/6/docs/technotes/tools/windows/classpath.html[Oracle documentation,window=_blank].
+
+==== CP Data Structures
+
+You can migrate the following CP data structures:
+
+* xref:data-structures:iatomiclong.adoc[]
+* xref:data-structures:iatomicreference.adoc[]
+* xref:data-structures:icountdownlatch.adoc[]
+
+If a CP data structure has no data, it is not migrated. For example, where the `set()` method has not been called on an existing `AtomicReference`. This is because such data structures are created lazily and do not exist in the cluster until used. This avoids the creation of a server-side resource with null data.
+
+To configure the  migration of CP data structures, you must update the _config.json_ file located in the _migration_config_ folder.
```

Comment:
```suggestion
To configure the migration of CP data structures, you must update the _config.json_ file located in the _migration_config_ folder.
```

we'll not have an option about cp structures I think.  so we can move this sentence to a different place and say:

To configure the data migration, you must update the _config.json_ file located in the _migration_config_ folder.

Reply:
It might be reasonable to keep this PR open until we finish the config.json addition then. Not to have to open a new pr